packages/firestore/test/unit/remote/bloom_filter.test.ts

```diff
@@ -198,25 +198,34 @@ describe('BloomFilter', () => {
         TEST_DATA.count5000Rate0001TestResult
       );
     });
-    it('mightContain result for 50000 documents with 1 false positive rate', () => {
+
+    // Skip the large sized golden tests below as they are flaky on CI.
+    // These tests can be run manually if needed.
+    it.skip('mightContain result for 50000 documents with 1 false positive rate', () => {
```

Comment:
You have to add a comment

```
// eslint-disable-next-line no-restricted-properties
```
above the line that uses `it.skip()` because it is a "restricted property"

e.g. https://github.com/firebase/firebase-js-sdk/blob/43e402fb49a081a59729290627c7b20099ca46a4/packages/firestore/test/lite/integration.test.ts#L772-L773